src/components/Logo/logo.svg

```diff
@@ -1,58 +0,0 @@
-		c-5-2.9-10-4.3-15-4.3s-10,1.5-15,4.3c-16.5,9.5-29.2,32.3-36.8,49.8l-0.1,0.3l-0.3,0c-2.4,0.3-4.3,0.5-6.6,0.8l-0.8,0.1
-		L-60.1,163.5z"/>
-	<path class="st3" d="M67.5,214.7c-1.8,0-3.7-0.1-5.5-0.2l-0.6,0l-0.1-0.6c-1.6-12.8-5.3-27.9-10.5-42.3l-0.4-1.1l1.1,0.1
-		c2.1,0.3,4.1,0.6,6.1,0.9l0.4,0.1l0.1,0.4c4.9,14.4,8.4,29.3,9.9,41.9l0.1,0.8L67.5,214.7z"/>
-	<path class="st1" d="M61.5,182.1c-1-3.4-2.1-6.8-3.2-10.1l-0.1-0.4l-0.4-0.1c-1.9-0.3-4-0.6-6.1-0.9l-1.1-0.1l0.4,1.1
-		c1.1,3.2,2.2,6.4,3.2,9.7C57.7,181.6,59.7,181.9,61.5,182.1z"/>
-	<path class="st3" d="M-69.5,213.6c1.5-12.6,5-27.5,9.9-41.9l0.1-0.4l0.4-0.1c1.9-0.3,4-0.6,6.1-0.9l1.1-0.1l-0.4,1.1
-		c-5.1,14.5-8.8,29.5-10.5,42.3l-0.1,0.6l-0.6,0c-1.8,0.1-3.7,0.2-5.5,0.2l-0.8,0L-69.5,213.6z"/>
-	<path class="st1" d="M-59,171.2l-0.4,0.1l-0.1,0.4c-1.1,3.4-2.2,6.8-3.2,10.2c3.4-0.4,4.4-0.4,7.3-0.7c1-3.3,2.1-6.6,3.3-9.9
-		l0.4-1.1l-1.1,0.1C-55,170.6-57.1,170.9-59,171.2z"/>
-	<path class="st3" d="M-0.9,300c-10.1,0-20-2-29.3-6c-9-3.9-17.1-9.5-23.9-16.5c-6.8-7-12-15.2-15.6-24.2c-3.7-9.5-5.4-19.6-5-30.1
-		l0-0.5l0.5,0c2.4-0.1,4.1-0.1,5.8-0.2l0.5,0l0,0.5c-0.4,9.7,1.1,18.7,4.5,27.5c3.2,8.3,8,15.7,14.1,22.2
-		c6.2,6.5,13.5,11.5,21.7,15.1c8.4,3.7,17.4,5.5,26.7,5.5c9.2,0,18.2-1.9,26.6-5.5c8.2-3.6,15.5-8.6,21.7-15.1
-		c6.2-6.4,10.9-13.9,14.1-22.1c3.4-8.8,4.9-17.8,4.5-27.5l0-0.5l0.5,0c2.4,0.1,4,0.2,5.8,0.2l0.5,0l0,0.5
-		c0.4,10.5-1.3,20.6-5.1,30.1c-3.6,9.1-8.8,17.2-15.6,24.3c-6.8,7.1-14.9,12.6-23.9,16.5C19.1,297.9,9.3,300-0.9,300z"/>
-	<path class="st1" d="M-67.7,231.8c-0.2-2.9-0.3-5.8-0.2-8.9l0-0.5l-0.5,0c-1.7,0.1-3.4,0.2-5.8,0.2l-0.5,0l0,0.5
-		c-0.1,3,0,6,0.2,8.9l0.2,0C-72,231.9-69.8,231.9-67.7,231.8L-67.7,231.8z"/>
-	<path class="st1" d="M72.9,222.6l-0.5,0c-1.7,0-3.4-0.1-5.8-0.2l-0.5,0l0,0.5c0.1,3,0.1,5.9-0.2,8.8l6.8,0.3
-		c0.2-2.9,0.3-5.9,0.2-8.9L72.9,222.6z"/>
-	<path class="st1" d="M76.7,224c-14.9,0-29-1.8-41.8-5.3c-6.5-1.7-12.5-4-17.1-6.2l-0.1-0.1c-11.5-5.5-12-16.6-10.9-26.6
-		c0.3-2.5-0.7-5.9-3.3-7.7c-1.3-0.8-2.7-1.3-4.2-1.3c-1.5,0-3,0.4-4.2,1.3c-2.7,1.8-3.6,5.1-3.3,7.7c1.1,10.1,0.6,21.1-10.9,26.7
-		l-0.1,0.1c-4.6,2.2-10.7,4.5-17.1,6.2c-12.9,3.5-27,5.3-41.8,5.3h-0.1c-5.8,0-10.9-4-12.4-9.7c-2.2-8.5-5.1-21.1-5.4-33
-		c-0.1-4.8,2.6-9.1,6.9-11.1c15.9-7.1,48-9.7,61.8-9.5c5,0.1,9.4,1.9,12.7,5.3c4.7-2.1,9.4-3.2,14-3.2c4.6,0,9.3,1.1,14,3.2
-		c2.3-2.4,6.4-5.2,12.7-5.3c8.2-0.1,43.6,1.4,61.8,9.5c4.4,1.9,7.1,6.3,6.9,11c-0.3,11.9-3.3,24.6-5.4,33
-		C87.7,220,82.6,224,76.7,224L76.7,224z M27.6,171.4c-6.7,0.1-8,5.7-8.2,6.9c-0.6,3.4-1.4,8.6-1.7,14.2c-0.1,5.4,1.9,9.2,5.8,11.1
-		c14.5,6.5,31.4,9.7,51.6,9.8c1.6,0,3-1,3.3-2.6c1.9-7.5,4.5-18.7,4.8-28.9c0-1-0.5-1.9-1.4-2.3C67.9,173.3,38.4,171.2,27.6,171.4z
-		 M-83.4,179.5c-0.9,0.4-1.5,1.3-1.4,2.3c0.3,10.2,2.9,21.4,4.8,28.9c0.4,1.5,1.8,2.6,3.3,2.6c20.2-0.1,37.1-3.3,51.7-9.8
-		c3.9-1.9,5.8-5.6,5.8-11.1c-0.2-4.3-0.8-9.3-1.7-14.2c-0.2-1.1-1.6-6.8-8.2-6.9C-35.6,171.3-67.8,172.5-83.4,179.5z"/>
-</g>
-</svg>
```

Comment:
Removing this for now as it was unused and also doesn't build (.svg files don't build with library currently)